.lintstagedrc.json

```diff
@@ -1,4 +1,5 @@
 {
+  "bin/lint-staged": ["prettier --write", "npm run lint:base -- --fix"],
```

Comment:
Maybe better rename the file to `*.js`? I don't think it makes any difference since it's referred in the `package.json`

Reply:
Good call, done.